exercises/practice/grains/.meta/config.json

```
@@ -0,0 +1,21 @@
+{
+  "blurb": "Calculate the number of grains of wheat on a chessboard given that the number on each square doubles.",
+  "authors": [
+  ],
+  "contributors": [
+    "glennj"
+  ],
```

You should be the author here. An author of an exercise is the one that added it to the track. I'm assuming you didn't want to claim authorship due to the exercise being defined in the prob-specs repo, but you're still the author as far as Exercism is concerned.

```suggestion
  "authors": [
      "glennj"
  ],
  "contributors": [
  ],
```

---

I assume this also needs to be updated for the other exercises. Just to ensure you get the right credits and reputation :)